### DISCUSSION TITLE
Can I know why `TransformStrategies` was deleted?

### DISCUSSION THREAD

**Discussion by kjyeon1676** [iree-org/iree#19572] [Q&A] [answered]
Hi, I remeber seeing the code that uses `Transform Dialect` and `TransformStrategies` (iree/compiler/Codegen/TransformStrategies/) for it. But, one day I found out that the related code was dropped with this PR: https://github.com/iree-org/iree/pull/18820
Can i know the RFC, docs, or reason related to the decision?

**Comment by ScottTodd** [comment, marked as answer]
Tagging the PR author: @qedawkins (you might also want to comment on the PR instead of opening a discussion)
